docs/7.Scan Examples/Dockerfile.md

```diff
@@ -48,30 +48,30 @@ Passed checks: 3, Failed checks: 2, Skipped checks: 0
 Check: CKV_DOCKER_5: "Ensure update instructions are not use alone in the Dockerfile"
 	PASSED for resource: /Dockerfile.
 	File: /Dockerfile:1-8
-	Guide: https://docs.prismacloud.io/en/enterprise-edition/policy-reference/docker-policies/docker-policy-index/ensure-update-instructions-are-not-used-alone-in-the-dockerfile
+	Guide: https://docs.prismacloud.io/en/enterprise-edition/policy-reference/docker-policies/docker-policy-index/docker-policy-index/ensure-update-instructions-are-not-used-alone-in-the-dockerfile
```

Comment:
Hi @tyutyutyu, thanks for your contribution.
the new URLs are not wrong, any reason you'd like to update?